### PR TITLE
More exception in analysis pipeline

### PR DESCRIPTION
**Pull request by dingquanyu**
1. add more exception catches in both run_get_good_pae.py and pdb_analyser.py
2. fixed the typo of Hydrophobhic column name before writing the output csv

### CODE REVIEW

File: alphapulldown/analysis_pipeline/get_good_inter_pae.py [hunk 1]
```diff
@@ -108,50 +108,67 @@ def main(argv):
...
+                    except Exception as e:
+                        logging.error(f"Error getting the best model name from ranking_debug.json for job :{job} : {e}")
+                except Exception as e:
+                    logging.error(f"Error processing ranking_debug.json for job {job}: {e}")
+            else:
+                logging.warning(f"{job} does not have ranking_debug.json. Skipped.")
+        except Exception as e:
+            logging.error(f"Error processing job {job}: {e}")
+        finally:
+            logging.info(f"done for {job} {count} out of {len(jobs)} finished.")
```
Comment (DimaMolod) [inline alphapulldown/analysis_pipeline/get_good_inter_pae.py:159]: A flat structure would be easier to read/comprehend than nested exceptions, but with 'finally' block at the end I think logic stays the same

File: alphapulldown/analysis_pipeline/get_good_inter_pae.py [hunk 2]
```diff
@@ -108,50 +108,67 @@ def main(argv):
...
+
     if len(good_jobs) == 0:
         logging.info(
             f"Unfortunately, none of your protein models had at least one PAE on the interface below your cutoff value : {FLAGS.cutoff}.\n Please consider using a larger cutoff.")
     else:
-        unwanted_columns = ['pi_score','pdb',' pvalue']
+        unwanted_columns = ['pdb',' pvalue', 'pvalue']
```
Comment (DimaMolod) [inline alphapulldown/analysis_pipeline/get_good_inter_pae.py:165]: Do you have 'pvalue' in unwanted columns defined twice intentionally?

**Review by DimaMolod — APPROVED** [review]
style is not perfect, but most importantly this version seems to be robust enough

File: alphapulldown/analysis_pipeline/get_good_inter_pae.py (hunk 2 above)
Comment (dingquanyu) [reply alphapulldown/analysis_pipeline/get_good_inter_pae.py:165]: > Do you have 'pvalue' in unwanted columns defined twice intentionally?

Yea it's quite annoying that I found in the PI score output files, some files write column name 'pvalue' but some write ' pvalue' with an empty space before letter p. I added twice just to be 100% sure I remove the pvalue column, regardless of the formats. It's a bit ugly solution I agree

**Pull request merged**